corehq/apps/hqadmin/reports.py

```diff
@@ -240,15 +240,8 @@ def rows(self):
             ]
 
     def _users_query(self):
-        query = (user_es.UserES()
-                 .remove_default_filters()
-                 .OR(user_es.web_users(), user_es.mobile_users()))
```

Comment:
I think this might actually be necessary to filter out unknown users

Reply:
https://github.com/dimagi/commcare-hq/pull/33694/commits/1769f9f111bb281a222a4972b86581fb2abbf98f